apps/api/src/app/inbox/dtos/session-request.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { IsDefined, IsOptional, IsString } from 'class-validator';
+
+export class SessionRequestDto {
```

Comment:
no swagger annotations because it won't be exposed as public API

Reply:
Should we maybe call it `SubscriberSessionRequestDto`? to make it clear is for subscribers and not users?